src/tree/decorators.ts

```diff
@@ -0,0 +1,90 @@
+import {
+  Component,
+  DebugElement,
+} from '@angular/core';
```

Comment:
@clbond if it was already there we can remove it under a different issue.
In this case we can either us creating our own type for this, or just use `any`.